std/typecons.d

```diff
@@ -4755,7 +4755,7 @@ if (!is(T == class) && !(is(T == interface)))
             import core.stdc.stdlib : malloc;
             import std.conv : emplace;
 
-            _store = cast(Impl*)malloc(Impl.sizeof);
+            _store = () @trusted { return cast(Impl*)malloc(Impl.sizeof); }();
```

Comment:
manual memory management is safe as long as you properly pair the allocation with the deallocation. This is exactly what RefCounted does.